src/EA.Weee.Integration.Tests/Handlers/GetAatfSummaryRequestHandlerTests.cs

```diff
@@ -142,6 +144,101 @@ private static List<NoteTonnage> NoteTonnages()
             };
         }
 
+        [Component]
+        public class WhenIGetAnAatfsSummaryDataIGetCorrectSummaryNotesCount : GetAatfSummaryRequestHandlerTestBase
+        {
+            private readonly Establish context = () =>
+            {
+                LocalSetup();
+
+                var organisation = OrganisationDbSetup.Init().Create();
+                OrganisationUserDbSetup.Init().WithUserIdAndOrganisationId(UserId, organisation.Id).Create();
+                var aatf = AatfDbSetup.Init().WithOrganisation(organisation.Id).Create();
+                var complianceYear = fixture.Create<int>();
+
+                // 10 draft note
+                EvidenceNoteDbSetup.Init().WithAatf(aatf.Id).WithOrganisation(organisation.Id).WithComplianceYear(complianceYear).Create();
```

Comment:
Create some notes with +1 -1 compliance year, this will mean the asserts confirm that they are not included